spring-core/src/test/java/facade/BookingFacadeTest.java

```diff
@@ -0,0 +1,174 @@
+package facade;
+
+import model.Event;
+import model.Ticket;
+import model.User;
+import org.assertj.core.util.Lists;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import service.EventService;
+import service.TicketService;
+import service.UserService;
+
+import java.util.Date;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class BookingFacadeTest {
+
+    @Mock UserService userService;
```

Comment:
for mock fields you can add suffix mock for understadble it's real object or mock, `userServiceMock `or `userServiceSpy`